### PR TITLE
fix: filter button text

### PR DESCRIPTION
**Pull request by johnkim-det**
## Description

Minor fix for spacing between words in Filter button

## Test Plan

Filter and Sort buttons should have equal spacing between words

Screen Shot 2023-08-14 at 5 54 44 PM

## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

File: webui/react/src/components/FilterForm/TableFilter.tsx [hunk 1]
```diff
@@ -69,7 +69,7 @@ const TableFilter = ({
         open={isOpenFilter}
         onOpenChange={onIsOpenFilterChange}>
         <Button hideChildren={isMobile} icon={<Icon decorative name="filter" />}>
-          Filter{fieldCount > 0 && <span>({fieldCount})</span>}
+          Filter {fieldCount > 0 && `(${fieldCount})`}
```
Comment (keita-determined) [inline webui/react/src/components/FilterForm/TableFilter.tsx:72]: maybe using fragment instead of template literal? up to you

**Review by keita-determined — APPROVED** [review]

**Comment by keita-determined** [comment]
Looks like the space sizes look equal in latest-main

**Comment by johnkim-det** [comment]
referring to space between "Filter" and "(1)"

**Comment by keita-determined** [comment]
> referring to space between "Filter" and "(1)"

got it. still LGTM

**Pull request merged**